css-brochure-form/styles.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Brandon Grotesque', sans-serif;
+}
+
+fieldset {
+  max-width: 400px;
+  height: 700px;
+  margin-right: auto;
+  margin-left: auto;
+  border: none;
+}
+
+legend {
+  text-align: center;
+  color: rgba(236, 93, 87, 255);
+  font-size: 1.5em;
+  font-weight: lighter;
+  letter-spacing: 2px;
+}
+
+input,
+select {
+  border-radius: 7px;
+  display: inline-block;
+  justify-content: stretch;
+  border-color: lightgray;
+  height: 70px;
+  width: 900px;
+  font-size: 2rem;
+  padding-left: 15px;
+}
+
+select {
+  background-color: #f3f3f3;
+}
+
+input {
+  margin: 0 0 50px 0;
+}
+
+button {
+  justify-content: flex-end;
```

Comment:
This can be removed. Since this is not a flex container, it is not having any effect.